Makefile

```diff
@@ -1,13 +1,48 @@
-TAG          = $(if $(CIRCLE_TAG),$(CIRCLE_TAG),PRTEST)
+BUILD_ARCH  := $(shell uname -m)
+BUILD_OS    := $(shell uname -s | tr '[:upper:]' '[:lower:]')
+TAG          = $(shell VER=$$(git tag | tail -1); echo $${VER:1})
```

Comment:
i don't think this is going to work in jenkins because of the way it "clones"

Reply:
There is a `git fetch --tags` that is executed prior to the `TAG` variable being evaluated, so this works as expected. Here's an example from my testing - https://jenkins.use.int.logdna.net/blue/organizations/jenkins/logdna%2Flogspout/detail/PR-41/67/pipeline/

Reply:
for  PR builds this TAG will have default val?
release dry run might be useful as part of CI test.

Reply:
The only stage that executes currently for PR builds is the lint stage, so this shouldn't ever be a problem.